extensions/service_bindings_manager.go

```diff
@@ -0,0 +1,76 @@
+package extensions
+
+import (
+	"os"
+	"path/filepath"
+)
+
+// ServiceBinding is a representation of a service binding that has
+// been provided into the build container.
+type ServiceBinding struct {
```

Comment:
We should consider using `libcnb` where we can. In this case, I think using their binding type would be good: https://github.com/buildpacks/libcnb/blob/main/platform.go#L52

This could get us on a path to eliminate at least a tiny bit of the overhead if we ever try to merge packit with libcnb and/or libpak